.github/workflows/deploy.yaml

```diff
@@ -205,8 +249,15 @@ jobs:
 
       - name: Building Docker image using Google Cloud Build
         run: | # TODO - actually parse semver to tag v1
+          # need to pull changes because we've updated the package versions in
```

Comment:
Just a suggestion: can we hide all this machinery inside `.sh` scripts?

Reply:
this parts needs more improvements which I'd like to do separately, but I generally agree